tracing-opentelemetry/src/layer.rs

```diff
@@ -273,7 +273,17 @@ impl<'a> field::Visit for SpanAttributeVisitor<'a> {
         }
 
         self.record(Key::new(field.name()).string(value.to_string()));
-        self.record(Key::new(format!("{}.chain", field.name())).array(chain));
+        self.record(Key::new(format!("{}.chain", field.name())).array(chain.clone()));
```

Comment:
i'm a little unsure—what's the difference between the chain and the `exception.stacktrace`?

Reply:
It's from the other PR, chain == Error's `source` method